applications/FluidDynamicsApplication/tests/test_FluidDynamicsApplication.py

```diff
@@ -60,8 +50,12 @@ def AssambleTestSuites():
     smallSuite.addTest(DarcyChannelTest('testDarcyDensity'))
     smallSuite.addTest(EmbeddedReservoirTest('testEmbeddedReservoir2D'))
     smallSuite.addTest(EmbeddedReservoirTest('testEmbeddedSlipReservoir2D'))
-    smallSuite.addTest(EmbeddedAusasCouetteTest('testEmbeddedAusasCouette2D'))
-    smallSuite.addTest(EmbeddedAusasCouetteTest('testEmbeddedDevelopmentCouette2D'))
+    smallSuite.addTest(EmbeddedCouetteTest('testEmbeddedCouette2D'))
```

Comment:
I recommend using `smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([EmbeddedReservoirTest]))`

this detects all the tests in `EmbeddedReservoirTest` automatically and they don't have to be added manually in this file

It happened several times in StructuralMechanics that developers wrote their tests but forgot to add them

Reply:
@philbucher thx for the advise! However, I guess that this functionality adds all the tests in a file to a unique suite (please correct me if I'm wrong). The point is that in our case some of them go to one suite and some of them to another.

Reply:
I think you are right, in this case you have to add it manually afaik